ncm-metaconfig/src/main/metaconfig/limits_conf/pan/config.pan

```diff
@@ -0,0 +1,8 @@
+unique template metaconfig/limits_conf/config;
+
+include 'metaconfig/limits_conf/schema';
+
+bind "/software/components/metaconfig/services/{/etc/security/limits.d/99-quattor.conf}/contents" = limits_conf_file;
```

Comment:
IMHO 99-quattor is too limited. What if some other system wants to deliver an override? This should go one higher than the OS default, or somewhere with some leeway for future expansion.